mcrcon.c

```diff
@@ -304,7 +304,7 @@ int net_connect(const char *host, const char *port)
 {
 	int sd;
 
-	struct addrinfo hints = {0};
+	struct addrinfo hints = {.ai_family = 0};
```

Comment:
I assume this change is due to compiler giving some grief about "missing initializer". Am I correct?

If I recall correctly some compilers are overly paranoid about it even though initializing with {0} should be idiomatic way of saying "initialize the whole thing with zeros" in C.

I think it better replace this line with memset(&hints, 0, sizeof hints); which should do the right thing and get rid of the warnings on all compilers.